src/graph/util/ExpressionUtils.cpp

```diff
@@ -177,6 +180,30 @@ Expression *ExpressionUtils::rewriteParameter(const Expression *expr, QueryConte
   return graph::RewriteVisitor::transform(expr, matcher, rewriter);
 }
 
+Expression *ExpressionUtils::rewriteInnerInExpr(const Expression *expr) {
```

Comment:
Why do this?

Reply:
Equivalently rewrite IN expression with only one operand into EQ expression to reuse index selection logic.